projects/storefrontlib/src/cms-components/myaccount/payment-methods/payment-methods.component.ts

```diff
@@ -35,6 +35,8 @@ export class PaymentMethodsComponent implements OnInit {
       })
     );
 
+    this.paymentMethods$.subscribe(); // Prevents flicker when deleting default payment (GH-5968)
+
```

Comment:
Can accomplish this while keeping the subscription in the template? That would align better with our best practices. 

We could essentially wrap the whole template in and ng-container:
```
<ng-container *ngIf="paymentMethods$ | async as paymentMethods">
...
</ng-container>
```

Reply:
Thanks for the suggestion, I forgot we could subscribe in the template